hack/e2e/environment.sh

```diff
@@ -22,6 +22,7 @@ export CAPI_VERSION=${CAPI_VERSION:-"v1beta1"}
 export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export NUM_NODES=${NUM_NODES:-"4"}
 export UPGRADE_TEST=${UPGRADE_TEST:-false}
+export EPHEMERAL_TEST=${EPHEMERAL_TEST:-false}
```

Comment:
I think we don't need to have the variable in this file. If I recall correctly, the only reason we have `UPGRADE_TEST` here is because we set CAPI/CAPM3 versions depending on if it is an upgrade test or not. But the file should basically only be for metal3-dev-env, which does not know about this `EPHEMERAL_TEST` variable.